### PR TITLE
[web] improve jest config, fix test and add more tests

### PR DESCRIPTION
**Pull request by AugustoCalaca**
before:
Captura de tela de 2019-10-28 23-50-27

after:
Captura de tela de 2019-10-28 23-50-55

### CODE REVIEW

File: packages/web/jest.config.js [hunk 1]
```diff
@@ -1,20 +0,0 @@
-module.exports = {
-  notify: false,
-  bail: false,
-  snapshotSerializers: ['jest-serializer-html'],
-  displayName: 'web-app',
-  setupFilesAfterEnv: [require.resolve('./jest.setup.js')],
-  transformIgnorePatterns: ['node_modules/'],
-  coverageReporters: ['lcov', 'html'],
-  transform: {
-    '^.+\\.tsx?$': 'ts-jest',
```
Comment (sibelius) [inline packages/web/jest.config.js]: I think the right fix is to use babel-jest here, instead of ts-jest

File: packages/web/test/babel-jest.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+const babelJest = require('babel-jest');
+
+ module.exports = {
```
Comment (sibelius) [inline packages/web/test/babel-jest.js]: is this needed?

File: packages/web/src/__tests__/UserList.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                name: "Bangkit Ilham",
+              }
+            }
+          ]
+        }
+      })
+    );
+
+    // debug();
+    expect(baseElement).toMatchSnapshot();
```
Comment (sibelius) [inline packages/web/src/__tests__/UserList.spec.tsx]: avoid snapshot

prefer a test like this

expect(getByText('Save')).toBeTruthy();

File: packages/web/src/__tests__/UserList.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import UserListDefault from '../UserList';
+import Environment from '../relay/Environment';
+
+afterEach(cleanup);
+
+describe('<UserList />', () => {
+  it('should return most recent operation', () => {
+    render(<UserListDefault />);
+    const operation = Environment.mock.getMostRecentOperation();
+    expect(operation.fragment.node.name).toBe('UserListQuery');
```
Comment (sibelius) [inline packages/web/src/__tests__/UserList.spec.tsx]: this is not testing much, you are just testing the mock payload is working well, not our component

File: packages/web/src/__tests__/UserList.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  });
+
+  it('should have pending operations in the queue', () => {
+    render(<UserListDefault />);
+    expect(Environment.mock.getAllOperations().length).toEqual(1);
```
Comment (sibelius) [inline packages/web/src/__tests__/UserList.spec.tsx]: not needed

File: packages/web/src/__tests__/UserList.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    const { debug, getByText, baseElement } = render(<UserListDefault />);
+    expect(getByText('loading')).toBeInTheDocument();
+
+    // debug();
+    Environment.mock.resolveMostRecentOperation((operation) =>
+    MockPayloadGenerator.generate(operation),
+    );
+
+    // debug();
+    expect(baseElement).toMatchSnapshot();
```
Comment (sibelius) [inline packages/web/src/__tests__/UserList.spec.tsx]: avoid snapshot

File: packages/web/src/__tests__/UserList.spec.tsx (hunk 3 above)
Comment (AugustoCalaca) [reply packages/web/src/__tests__/UserList.spec.tsx]: nice tip

File: packages/web/src/__tests__/UserList.spec.tsx (hunk 6 above)
Comment (AugustoCalaca) [reply packages/web/src/__tests__/UserList.spec.tsx]: improved

File: packages/web/src/__tests__/UserList.spec.tsx (hunk 5 above)
Comment (AugustoCalaca) [reply packages/web/src/__tests__/UserList.spec.tsx]: removed

File: packages/web/src/__tests__/UserList.spec.tsx (hunk 4 above)
Comment (AugustoCalaca) [reply packages/web/src/__tests__/UserList.spec.tsx]: removed

File: packages/web/test/babel-jest.js (hunk 2 above)
Comment (AugustoCalaca) [reply packages/web/test/babel-jest.js]: removed

**Pull request merged**